### PR TITLE
Add methods in cart page and Adapt sanity tests with 1.7.2

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add methods in cart page and Adapt sanity tests with 1.7.2
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Sponsor company   | @PrestaSHopCorp
| How to test?      | Ci is :green_apple:

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
